README.md

```diff
@@ -1,2 +1,29 @@
-# libdependency
-Library for dependency-management related functionality
+# Libdependency
+
+This exists to assist buildpack authors construct code to retrieve new versions and their associated metadata.
+
+It also contains various helpers and actions to assist the overall dependencies workflow.
+
+## Integration
+
+### Retrieval
```

Comment:
I'd also like to see the other functionality available documented here

Reply:
some docs have been added